planty/users/views.py

```diff
@@ -16,3 +18,39 @@ def register(request):
     else:
         form = UserRegisterForm()
     return render(request, 'users/register.html', {'form': form})
+
+
+@login_required
+def profile(request, pk):
+    user_page_owner = User.objects.get(id=pk)
+    context = {
+        'user_page_owner': user_page_owner,
+    }
+    return render(request, 'users/profile.html', context)
+
+
+@login_required
+def update_profile(request):
+    if request.method == 'POST':
+        user_form = UserUpdateForm(request.POST, instance=request.user)
+        profile_form = ProfileUpdateForm(request.POST, request.FILES,
+                                         instance=request.user.profile)
+
+        if user_form.is_valid() and profile_form.is_valid():
+            user_form.save()
+            profile_form.save()
```

Comment:
I think you could do 
```python
profile_form.save(commit=False)  # doesn't save to DB, just writes form data to python instance
user_form.save()  # saves user and profile to DB (because of the signal)
```
because the signal will automatically save the profile too. Does this work? I think currently you hit the DB unnecessarily twice.